apify-docs-theme/src/theme/custom.css

```diff
@@ -1579,3 +1578,15 @@ html[data-theme='dark'] .runnable-code-block svg .apify-logo {
 .apiPage .tsd-panel-header .tsd-anchor-id {
     top: -13rem;
 }
+
+.youtubeVideoPlayer {
```

Comment:
If it is possible I would prefer a solution without the wrapper. Because someone can easily forget to use it when adding a new video. Maybe we can use this selector to target just the youtube frames `iframe[src*="youtube"]`

Reply:
Forget about this comment. It is in fact possible haha

Reply:
keep in mind youtube also uses other domains like youtu.be